test/functional/api/vim_spec.lua

```diff
@@ -1267,5 +1268,44 @@ describe('API', function()
       eq(expected, nvim("list_uis"))
     end)
   end)
-
+  describe('statusline', function()
+    it('last char shows (multibyte)', function()
+      local screen = Screen.new(20, 4)
+      screen:attach()
+      command('set laststatus=2')
+      command('set statusline=你好')
+      screen:expect([[
+      ^                    |
+      ~                   |
+      你好                |
+                          |
+      ]])
+    end)
+  end)
+  describe('statusline', function()
```

Comment:
Why not group the following tests with the previous `describe()` block?

Reply:
To be honest, i was not sure where to put them! Do you have an idea where the most fitting place would be?

Reply:
I would have grouped them with the previous `describe()` block, because of the same name `statusline`.

Reply:
Oh, that was just a copy paste error, i will correct it.